test/test.js

```diff
@@ -208,4 +208,19 @@ describe('postcss-css-variables', function() {
 	});
 
 
+	it('Should use string values for `undefined` values, see #22', function() {
+		return postcss([
+			cssvariables()
+		])
+		.process(fs.readFileSync('./test/fixtures/missing-variable-usage.css', 'utf8'))
+		.then(function(result) {
+			var root = result.root;
+			var fooRule = root.nodes[0];
+			expect(fooRule.selector).to.equal('.box-foo');
+			var colorDecl = fooRule.nodes[0];
+			expect(colorDecl.value).to.be.a('string');
```

Comment:
I didn't have this in my snippet but lets also add a `expect(colorDecl.value).to.be.equal('undefined');`